### DISCUSSION TITLE
アドベントカレンダー当日でもタイトルだけを入れて更新できるようにしてほしい

### DISCUSSION THREAD

**Discussion by mikecat** [increments/qiita-discussions#941] [要望・アイデア・不具合]
アドベントカレンダーにおいて、当日でもとりあえずタイトルだけを入れて (URLは空欄で) 更新できるようにしてほしいです。

記事を書くのが間に合わなそうだが、とりあえずタイトルだけ入れて書く意思はあることを示すのに使いたいです。

現状では、「URLを入力してください」と出てしまい、更新できなそうです。

**Comment by tanaka-asami415** [comment]
@mikecat 
リクエストありがとうございます！

> 当日でもとりあえずタイトルだけを入れて (URLは空欄で) 更新できるようにしてほしいです。

対応は来年以降になってしまいますが、URLが空欄でも更新できるようにしたいと思います。

現在、新規に登録する際はURLが空欄でも登録できるようになっているため、
「意思表示だけしておきたい！」という際は、お手数ですが一度参加を取り消していただいた上で、再度カレンダーの参加をしていただけると幸いです🙏
